### ISSUE TITLE
Support for Selenium 3.7.0

### ISSUE THREAD

**Issue by barclay-reg** [FluentLenium/FluentLenium#564]
Hi all,

currently FluentLenium 3.4.1 is incompatible to Selenium 3.7.x

The following exception is originated in some refactorings in the API about JSON (see https://github.com/SeleniumHQ/selenium/commit/6104872dc1424838466bfa065ad69121ad1f8bb8)

```
java.lang.NoClassDefFoundError: org/openqa/selenium/remote/JsonException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.fluentlenium.configuration.DefaultConfigurationFactory.newConfiguration(DefaultConfigurationFactory.java:39)
	at org.fluentlenium.configuration.ConfigurationFactoryProvider.getConfigurationFactory(ConfigurationFactoryProvider.java:24)
	at org.fluentlenium.configuration.ConfigurationFactoryProvider.newConfiguration(ConfigurationFactoryProvider.java:46)
	at org.fluentlenium.adapter.FluentAdapter.<init>(FluentAdapter.java:23)
	at org.fluentlenium.adapter.FluentTestRunnerAdapter.<init>(FluentTestRunnerAdapter.java:55)
	at org.fluentlenium.adapter.FluentTestRunnerAdapter.<init>(FluentTestRunnerAdapter.java:36)
	at org.fluentlenium.adapter.FluentTestRunnerAdapter.<init>(FluentTestRunnerAdapter.java:27)
	at org.fluentlenium.adapter.junit.FluentTest.<init>(FluentTest.java:15)
```

**Comment by filipcynarski** [comment]
Well they have introduced a bunch of breaking changes so it will be
addressed in next release ;)

W dniu sob., 11.11.2017 o 15:34 Robert Kleinschmager <
notifications@github.com> napisał(a):

**Comment by barclay-reg** [comment]
I can make a PR, as I needed to test something out here ...

Am 11. Nov. 2017, 17:16 +0100 schrieb Filip Cynarski <notifications@github.com>:

**Comment by martin-g** [comment]
Any idea when the next release (including) this upgrade will be ?

**Comment by filipcynarski** [comment]
Hello @martin-g I'm staring working again on FL, I was busy due to building my house last 2 months so that's why our works slowed down

I think I can change the scope of this release to let this update came out :)

**Comment by filipcynarski** [comment]
@martin-g I'm upgrading it to 3.8.1

**Comment by martin-g** [comment]
Awesome! Thank you @filipcynarski !

I hope you have built a nice house! :smile:

**Comment by gkiszczak** [comment]
@martin-g @barclay-reg  FluentLenium 3.5.0 version has been released today. Please update your dependencies.

**Comment by martin-g** [comment]
Selenium 3.9.0 has been released 2 days ago. It seems you didn't notice.
Or you updated to 3.8.1 intentionally ?

**Comment by martin-g** [comment]
Well, if there are no API break from Selenium side then I can manage this locally!
I will run my tests right away!

**Comment by filipcynarski** [comment]
I've released 3.5.1 version of FL which contains Selenium 3.9.0 sorry, I didn't notice.

**Comment by martin-g** [comment]
Thank you, @filipcynarski !
